Client/Ecosia/Settings/EcosiaSettings.swift

```diff
@@ -170,6 +170,27 @@ class EcosiaTermsSetting: Setting {
     }
 }
 
+class EcosiaSendAnonymousUsageDataSetting: BoolSetting {
```

Comment:
```suggestion
final class EcosiaSendAnonymousUsageDataSetting: BoolSetting {
```
Small suggestion mostly as good practice for us. Would be good if we'd make it the other ones `final` too as part of the change. Agree in case not to, as not part of this scope (just for good housekeeping 🏠 🧹 )

Reply:
Makes sense, always good to have housekeeping 😄 🧹 will update them all, thanks for pointing it!

Also thanks for the tech debt memory leak, great finding! Will also investigate it a bit to understand the effort needed 💧🔨